bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx115-eu-vpn.html

```diff
@@ -21,6 +21,8 @@
 {% endblock %}
 
 {% block experiments %}
+  {{ js_bundle('firefox_whatsnew_115_na_win7') }}
```

Comment:
Would this try to redirect win7 users who are viewing on a non-English locale? Should it be limited to `en` only?

Reply:
Is it worth also putting this behind a switch, incase we need to turn it off should something go awry?